hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/storage/ContainerProtocolCalls.java

```diff
@@ -687,12 +719,28 @@ public static void validateContainerResponse(
   }
 
   public static List<CheckedBiFunction> getValidatorList() {
-    List<CheckedBiFunction> validators = new ArrayList<>(1);
+    return VALIDATORS;
+  }
+
+  private static final List<CheckedBiFunction> VALIDATORS = createValidators();
+
+  private static List<CheckedBiFunction> createValidators() {
     CheckedBiFunction<ContainerProtos.ContainerCommandRequestProto,
         ContainerProtos.ContainerCommandResponseProto, IOException>
         validator = (request, response) -> validateContainerResponse(response);
+    return Collections.singletonList(validator);
+  }
+
+  public static List<CheckedBiFunction> getValidatorList(
```

Comment:
IMO a getXXX() method typically implies a O( 1 ) operation. It would be better off to rename it, such as toValidatorList() ?

Reply:
> ... a getXXX() method typically implies a O( 1 ) operation. ...

This is not true for TreeMap.get(..), LinkedList.get(..), etc.  But I am okay to do the rename.